data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/converter/StreamRecordConverter.java

```diff
@@ -63,6 +64,11 @@ public Event convert(final Map<String, Object> rowData,
 
         EventMetadata eventMetadata = event.getMetadata();
 
+        // Only set external origination time for stream events, not export
+        final Instant externalOriginationTime = Instant.ofEpochMilli(eventCreateTimeEpochMillis);
```

Comment:
is eventCreateTimeEpochMillis read from the bin logs ?

Reply:
Yes, it's from binlog event timestamp: https://github.com/opensearch-project/data-prepper/blob/d6465efc63f98380605b90ddeab90343bb469dbf/data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/stream/BinlogEventListener.java#L224